python/triton/runtime/driver.py

```diff
@@ -49,7 +51,7 @@ class DriverConfig:
 
     def __init__(self) -> None:
         self.default: LazyProxy[DriverBase] = LazyProxy(_create_driver)
-        self.active: LazyProxy[DriverBase] | DriverBase = self.default
+        self.active: Union[LazyProxy[DriverBase] | DriverBase] = self.default
```

Comment:
```suggestion
        self.active: Union[LazyProxy[DriverBase], DriverBase] = self.default
```

Reply:
Whoops, was typing to fast. Funny `mypy` didn't yell